### PR TITLE
Primitive List feature(Part6: OsList)

### PR DESCRIPTION
**Pull request by zaki50**

### CODE REVIEW

File: realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java [hunk 1]
```diff
@@ -635,7 +635,7 @@ public static String getTableName() {
             } else if (name.equals("columnMutableRealmInteger")) {
                 Long val = null;
                 if (reader.peek() != JsonToken.NULL) {
-                    val = reader.nextLong()
+                    val = reader.nextLong();
```
Comment (cmelchior) [inline realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java]: How did this compile before? 🤔

**Comment by zaki50** [comment]
@cmelchior I've asked @beeender to implement empty methods in this PR.
Could you review this again after that?

File: realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java (hunk 1 above)
Comment (zaki50) [reply realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java]: No idea.
I guess Google's `compile-testing` library does not see the semi-colon.

**Comment by zaki50** [comment]
Implementation of `OSList` is added in separate PR (Part 10, #5216)

Could you review this? @realm/java

**Pull request merged**